### PR TITLE
Changes for updated product_configs

### PR DESCRIPTION
**Pull request by adydychk**

### CODE REVIEW

**Comment by adydychk** [comment]
@azhogov @dlobanox Need to discuss the changes like in this commit. I want to know right way...

It is not used in our CI now because I implemented the workaround for `release` on the side of product_configs (`mv ./__bin/Release ./__bin/release`).

https://github.com/Intel-Media-SDK/product-configs/pull/14

File: bb/master/master.cfg [hunk 1]
```diff
@@ -46,7 +46,7 @@ def init_build_factory(conf_file="conf_linux_public.py",
                                                          conf_file=conf_file),
                                         "--root-dir", util.Interpolate(r"%(prop:builddir)s/build_dir"),
                                         "--changed-repo", util.Interpolate(config.REPO_INFO),
-                                        "--build-type", config.BUILD_TYPE,
+                                        "--build-type", config.BUILD_TYPE, #TODO: possible unnecessary option
```
Comment (adydychk) [inline bb/master/master.cfg]: @azhogov please, need to discuss it :)

File: bb/master/master.cfg [hunk 2]
```diff
@@ -34,7 +34,8 @@ class Stage(Enum):
     COPY = "copy"
 
 def init_build_factory(conf_file="conf_linux_public.py",
-                       product_type=config.MASTER_PRODUCT_TYPE):
+                       product_type=config.MASTER_PRODUCT_TYPE,
+                       api_version="current"):
```
Comment (azhogov) [inline bb/master/master.cfg]: should be api_latest=true

File: bb/master/master.cfg [hunk 3]
```diff
@@ -120,11 +122,16 @@ c["schedulers"] = [
...
+                                    api_latest="false")
 #Init build factory for other branches (all except master)
-build_not_master_factory = init_build_factory("conf_linux_public.py", config.MASTER_PRODUCT_TYPE)
-build_api_latest_factory = init_build_factory("conf_linux_public_api_latest.py",
-                                              "api_latest")
+build_not_master_factory = init_build_factory("conf_linux_public.py",
+                                               config.MASTER_PRODUCT_TYPE,
+                                               api_latest="false")
+build_api_latest_factory = init_build_factory("conf_linux_public.py",
+                                              "api_latest",
```
Comment (azhogov) [inline bb/master/master.cfg]: it seems strange

File: bb/master/master.cfg [hunk 4]
```diff
@@ -50,7 +51,8 @@ def init_build_factory(conf_file="conf_linux_public.py",
                                         "--build-event", "commit",
                                         "--product-type", product_type,
                                         "--repo-url", util.Interpolate(r"%(prop:repository)s"),
-                                        "--stage", stage.value],
+                                        "--stage", stage.value,
+                                        f"api_latest={api_latest}"],
```
Comment (azhogov) [inline bb/master/master.cfg]: the arg can be added in case of true only.

File: bb/master/master.cfg [hunk 5]
```diff
@@ -35,26 +35,44 @@ class Stage(Enum):
...
+                                            "--product-type", product_type,
+                                            "--repo-url", util.Interpolate(r"%(prop:repository)s"),
+                                            "--stage", stage.value,
+                                            f"api_latest=True"],
+                                   workdir=r"../infrastructure/build_scripts",
+                                   name=stage.value))
+    else:
+        for stage in Stage:
+            build_factory.addStep(
+                steps.ShellCommand(command=[config.RUN_COMMAND,
```
Comment (azhogov) [inline bb/master/master.cfg]: please don't duplicate code
you can just append api_latest if it was set

**Pull request merged**